staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -965,6 +965,22 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation.
 	// +optional
 	AllowPrivilegeEscalation bool `json:"allowPrivilegeEscalation,omitempty" protobuf:"varint,16,opt,name=allowPrivilegeEscalation"`
+	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
```

Comment:
These comments are used to generate user-facing documentation and should not reference Go field names and types.